<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorVibrato</class>
 <widget class="QWidget" name="InspectorVibrato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>129</height>
   </rect>
  </property>
  <property name="accessibleName">
   <string>Vibrato Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Vibrato</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="panel" native="true">
     <layout class="QGridLayout" name="layout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Placement:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>placement</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="placement">
        <property name="accessibleName">
         <string>Placement</string>
        </property>
        <item>
         <property name="text">
          <string>Above</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Below</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="playArticulation">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>vibratoType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="vibratoType">
        <property name="accessibleName">
         <string>Type</string>
        </property>
        <item>
         <property name="text">
          <string>Vibrato</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vibrato Wide</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vibrato Sawtooth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vibrato Sawtooth Wide</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="Ms::ResetButton" name="resetVibratoType"/>
      </item>
      <item row="1" column="3">
       <widget class="Ms::ResetButton" name="resetPlacement"/>
      </item>
      <item row="2" column="3">
       <widget class="Ms::ResetButton" name="resetPlayArticulation"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ms::ResetButton</class>
   <extends>QWidget</extends>
   <header>inspector/resetButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>vibratoType</tabstop>
  <tabstop>placement</tabstop>
  <tabstop>playArticulation</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
